src/status_im/ui/screens/about_app/views.cljs

```diff
@@ -24,7 +24,7 @@
      {:type                :small
       :title               (i18n/label :t/version)
       :accessibility-label :version
-      :accessories         [(str app-version ";\n" node-version)]}]]])
+      :accessories         [(str app-version "; " node-version)]}]]])
```

Comment:
it's important to see full version, because there is no sense to see only part of version, so maybe we should use here custom component to show version , don't think we should tune list-item for that, or i've a better idea to show app-version and node-version in different items, that might work

Reply:
I like `app-version` and `node-version` being shown using separate list-item 👍 

It was like that in old design. 

The only reason why it is in one at the moment, is because it was specified in figma like that.

Reply:
@rachelhamlin can we please decide on the verbiage for each? "App version" and "Node version" seem obvious, but just want to make sure.

Reply:
Oh and tuning `list-item` is not because of this specific case, but in general. I saw that in `Profile-tab>Sync settings>Mailserver` in iPhone X or Xs, if the chosen mailserver happens to be one of the long named ones, it squishes the `title`. I figured, generally it is good to have some whitespace separating `title` and `long text accessory`. cc @errorists.

Note: (personal observation) In latest `transaction popup` design whitespace between Title/accessories seems to be cutting close. I'm not sure what is going to happen in smaller width devices. cc @andmironov @errorists

Reply:
Great point @bitsikka regarding the list item in transaction overview modals. I'll add a rule for that (a fixed-width minimum margin between title and accessory + truncate the values if necessary)

Reply:
<img width="585" alt="Screen Shot 2019-09-05 at 11 32 34" src="https://user-images.githubusercontent.com/3115606/64325472-e134ec80-cfd0-11e9-9663-ee2f8a574761.png">

Reply:
@bitsikka to make sure I understand context, are you speaking of the "About" screen within profile? I am happy with "App version" and "Node version" if so. I see in [Figma](https://www.figma.com/file/bPS9GrvMr7LnH7vnkmuveQfd/Settings?node-id=415%3A153) that they are combined into one line. I'm happy either way, but would probably prefer separate items as well.

Reply:
@rachelhamlin Yes, that is the one. Sounds good! separate items it is 👍

Reply:
@andmironov I tried to handle the edge case automatically in the component but it got too complicated. To keep things simple, I went with what I had done already(bearing in mind we have alternative provision for edge cases already):

- Long stringified accessory has max-width of 62% of device width. Btw, `Node version` is still too long in a separate line of its own.
- That means `title` is also constrained to not be longer than 30ish%(considering hard right-margin in `title` of 16px when there is accessory)
- This will handle simple cases like the long `Mailserver` name gracefully enough.
- This does not mean edge case like the `transaction modal` has to confirm to that rule.
- For edge cases like that, an implementer can choose to use `component` as `accessory` and/or `title`. In which case there is still maximum freedom outside the bounds of `title`/`accessories` area of list-component.
- That means we are leaving it up to the implementer of edge-case like scenario to conform to edge-case spec you just made.

Note: It will be best also for designers of verbiage that go in list-item and elsewhere to be mindful about smaller width devices.